src/devtools/useGotoAtomsSnapshot.ts

```diff
@@ -13,9 +22,17 @@ export function useGotoAtomsSnapshot(scope?: Scope) {
   }
 
   return useCallback(
-    (values: Iterable<readonly [Atom<unknown>, unknown]>) => {
-      store[RESTORE_ATOMS](values)
+    (values: Iterable<readonly [Atom<unknown>, unknown]> | AtomsSnapshot) => {
+      if (isIterable(values)) {
+        store[RESTORE_ATOMS](values)
```

Comment:
Let's add a deprecation message.
```suggestion
        if (__DEV__) {
          console.warn('snapshot as iterable is no longer supported. use an object instead.')
        }
        store[RESTORE_ATOMS](values)
```